dev/Breadcrumb/BreadcrumbBar.cpp

```diff
@@ -148,7 +148,11 @@ void BreadcrumbBar::UpdateItemsRepeaterItemsSource()
     if (ItemsSource())
     {
         m_breadcrumbItemsSourceView = winrt::ItemsSourceView(ItemsSource());
-
+        if (const auto& itemsRepeater = m_itemsRepeater.get())
+        {
+            m_itemsIterable = winrt::make_self<BreadcrumbIterable>(ItemsSource());
+            itemsRepeater.ItemsSource(*m_itemsIterable);
+        }
```

Comment:
I copied this over from the OnBreadcrumbBarItemsSourceCollectionChanged handler, why is the collection items changed handler even setting the ItemSource? Was this intentional or just an oversight?